tests/designs/sample_module/Makefile

```diff
@@ -51,5 +51,5 @@ else
     $(error "A valid value (verilog or vhdl) was not provided for TOPLEVEL_LANG=$(TOPLEVEL_LANG)")
 endif
 
-include $(COCOTB)/makefiles/Makefile.inc
-include $(COCOTB)/makefiles/Makefile.sim
+include $(shell cocotb-config --makefiles)/Makefile.inc
+include $(shell cocotb-config --makefiles)/Makefile.sim
```

Comment:
This change seems sort of out of place, but I suppose if CI passes it's fine.

Reply:
Yeah, that's an oopsie commit on me, will remove.